### PR TITLE
Updated bs 4.4.1

### PR DESCRIPTION
**Pull request by chauger**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,17 +6,19 @@
...
+    "bootstrap": "^4.4.1"
   },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "node-sass --output-style compressed --precision 6 --importer importer.js scss/style.scss dist/add-style.min.css",
     "build-docs": "node-sass --output-style expanded --source-map true --source-map-contents true --precision 6 --importer importer.js docs/style.scss docs/style.css",
     "watch": "nodemon --watch scss --ignore dist/ -e scss -x \"npm run build\""
+  },
+  "dependencies": {
+    "node-sass": "^4.13.1"
```
Comment (frodare) [inline package.json:22]: why was this moved out of dev dependencies?

Comment (chauger) [reply package.json:22]: did that on its own when I `npm i node-sass`
